<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab5sensordatabase</class>
 <widget class="QWidget" name="Tab5sensordatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="pDateTimeEditFrom">
       <property name="date">
        <date>
         <year>2025</year>
         <month>7</month>
         <day>16</day>
        </date>
       </property>
       <property name="time">
        <time>
         <hour>9</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="pDateTimeEditTo">
       <property name="date">
        <date>
         <year>2025</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBsearchDB">
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBdeleteDB">
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5">
     <item>
      <widget class="QTableWidget" name="pTBsensor">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>날짜</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>조도</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>온도</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>습도</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vertical" stretch="1,9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="pChartViewLayout"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
